File: method_overriding_python/Cylinder.py
from Circle import Circle 

class Cylinder(Circle):
        
    def __init__(self, radius, color, heigth):
        super().__init__(radius, color)
        self.heigth = heigth
        
    def getArea(self):
        self.area = 2 * 3.1416 * self.radius * self.heigth + 2 * super().getArea()
        return self.area 
    
        
        
        
        

File: method_overriding_java/Cylinder.java
public class Cylinder extends Circle{
    double height;

    public Cylinder(){
        this.height = 1.0;
    }

    public Cylinder(double radius, String color, double height){
        super(radius, color);
        this.height = height;
    }

    @Override
    public double getArea(){
        double area = 2 * 3.1416 * radius * height +  2 * super.getArea();
        return area;
    }
}